vng_api_common/audittrails/viewsets.py

```diff
@@ -19,6 +19,16 @@ def get_audittrail_main_object_url(self, data, main_resource):
             return data[self.audittrail_main_resource_key]
         return data[main_resource]
 
+    def get_applicatie_id(self):
+        try:
+            applicatie_id = self.request.jwt_payload['client_id']
+        except:
+            applicatie_id = self.request.META.get('X-NLX-Request-Application-Id', '')
```

Comment:
When/if we upgrade everything to Django 2.2: https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpRequest.headers